cloudify_cli/cli/cfy.py

```diff
@@ -1776,6 +1776,16 @@ def common_options(self, f):
             f = arg(f)
         return f
 
+    def local_common_options(self, f):
```

Comment:
the main point of this is that of course I'm going to be using it for many local commands soon.
But for now, just the one.